examples/gluon_alexnet/gluon_alexnet.py

```diff
@@ -0,0 +1,136 @@
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Flatten())
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+            self.output = nn.Dense(classes)
+
+    def forward(self, F, x):
+        x = self.features(x)
+        x = self.output(x)
+        return x
+
+
+class HybridAlexNet(gluon.HybridBlock):
+    """
+    Hybrid Block gluon model
+    """
+    def __init__(self, classes=1000, **kwargs):
+        super(HybridAlexNet, self).__init__(**kwargs)
+        with self.name_scope():
+            self.features = nn.HybridSequential(prefix='')
+            with self.features.name_scope():
+                self.features.add(nn.Conv2D(64, kernel_size=11, strides=4,
+                                            padding=2, activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Conv2D(192, kernel_size=5, padding=2,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Conv2D(384, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Flatten())
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
+            self.output = nn.Dense(classes)
+
+    def hybrid_forward(self, x):
+        x = self.features(x)
+        x = self.output(x)
+        return x
+
+
+class MMSImperativeService(MXNetVisionService):
+    """
+    Gluon alexnet Service
+    """
+    def __init__(self, model_name, model_dir, manifest, gpu=None):
+        super(MMSImperativeService, self).__init__(model_name, model_dir, manifest, gpu)
+        # Uncomment the following to create a purely imperative model of Alexnet. Comment other self.net blocks below.
+        self.net = ImperativeAlexNet()
+
+        # Uncomment the following to use pretrained model and comment other self.net blocks above and below.
```

Comment:
I think this is also very confusing, asking users to comment out...
Why not have clear, separate, examples for each use case?

Reply:
Came here to say these things. I want separate service files, one for each example. The code for each will be in the docs and discussed in-part.

Reply:
Addressed the comments in the docs.